packages/strapi-plugin-i18n/tests/graphql.test.e2e.js

```diff
@@ -0,0 +1,120 @@
+'use strict';
+
+// Helpers.
+const { createTestBuilder } = require('../../../test/helpers/builder');
+const { createStrapiInstance } = require('../../../test/helpers/strapi');
+const { createAuthRequest } = require('../../../test/helpers/request');
+
+const builder = createTestBuilder();
+let strapi;
+let rq;
+let graphqlQuery;
+let localeId;
+
+const recipesModel = {
+  attributes: {
+    name: {
+      type: 'string',
+    },
+  },
+  pluginOptions: {
+    i18n: {
+      localized: true,
+    },
+  },
+  connection: 'default',
+  name: 'recipes',
+  description: '',
+  collectionName: '',
+};
+
+describe('Test Graphql API create localization', () => {
+  beforeAll(async () => {
+    await builder.addContentType(recipesModel).build();
+
+    strapi = await createStrapiInstance();
+    rq = await createAuthRequest({ strapi });
+
+    graphqlQuery = body => {
+      return rq({
+        url: '/graphql',
+        method: 'POST',
+        body,
+      });
+    };
+
+    const locale = await strapi.query('locale', 'i18n').create({
```

Comment:
Why didn't you used `addFixtures()` (from the builder) instead of making this query manually (+ then deleting it manually in the afterAll)?